solutions/usaco-624.mdx

```diff
@@ -11,7 +11,9 @@ The
 [official solution](http://www.usaco.org/current/data/sol_balancing_platinum_feb16.html)
 uses a segment tree, which is introduced in Gold. However, we can just binary
 search on the answer.
+<LanguageSection>
```

Comment:
move time complexity under the explanation, and add ## Explanation and ## Implementation headers